crates/rolldown/src/ecmascript/format/iife.rs

```diff
@@ -56,7 +57,8 @@ pub fn render_iife(
     input_args
   ))));
 
-  if determine_use_strict(ctx) {
+  // Rollup also enables `use strict` in empty IIFE contents.
+  if determine_use_strict(ctx) || matches!(ctx.options.exports, OutputExports::None) {
```

Comment:
The rollup has a `output.strict` to control it, it's value default is true, because rollup only support esm. The rolldown is not need it, because rolldown support esm and cjs, it should be determine. 
We could add the option to do it if the user use it at future, i think it look like is unnecessary at now.

Reply:
Ok, I'll fix it now.